### ISSUE TITLE
Meteor 1.3 issue when building archive in xcode - segmentation fault: 11

### ISSUE THREAD

**Issue by vacarsu** [meteor/meteor#6965]
trying to build an archive for ios device to upload to itunes connect. I get the error below.
```
error: 'Cordova/CDV.h' file not found
#import <Cordova/CDV.h>
```

```xCode version: Version 7.3.1 (7D1014)```

**Comment by laosb** [comment]
Can you come up with a repro?

**Comment by vacarsu** [comment]
Deleting the build folder in .meteor/local fixed the issue.

**Comment by mwarren2** [comment]
I've just had this problem, which is pretty weird, but I found a solution.

It starts off with a warning saying: 
> Missing dependency target "CordovaLib (from CordovaLib.xcodeproj)"
This is strange because CordovaLib.xcodeproj is sitting right inside CordovaLib, so why doesn't it find it?

Then I got a string of 8 errors in files saying that they couldn't find **Cordova/CDV.h**:

> error: 'Cordova/CDV.h' file not found
> #import <Cordova/CDV.h>
> 

After several hours of trashing builds, rebuilding, restarting XCode and so on, I found an old page here with a solution which worked:
http://www.programmershare.com/2834908/

Basically you need to manually set a  **CORDOVALIB** setting to get things working again. 
Details are at the bottom of the article.

**Comment by JWPapi** [comment]
Is this really a good solution. What is the downside of deleting the local folder? If there are none why is it even included @vacarsu ?

**Comment by mwarren2** [comment]
@JWPapi There's no real downside to deleting the build folder. This can sometimes solve an odd problem, if it gets a bit full of gunge after lots of builds. 
However quite a bit of stuff is cached there, so the only downside is that the next build takes a bit longer than usual.

**Comment by JWPapi** [comment]
I just found out that after building a project the CordovaLib Path gets set inside the project, if you build a new version later and delete the old project folder, it wont find the Lib, so either meteor should install CordovaLib globally or replace the path with each project.

**Comment by qintengfei** [comment]
meteor reset the proj.then build the proj. open the xcode proj build it and there before got tips to translate old syntax to new swift syntax ,i select cancel.
